internal/tracing/mux.go

```diff
@@ -28,23 +29,45 @@ type MuxExporter struct {
 }
 
 func (m MuxExporter) ExportSpans(ctx context.Context, spans []sdktrace.ReadOnlySpan) error {
-	var eg multierror.Group
-	for i := range m.exporters {
-		exporter := m.exporters[i]
-		eg.Go(func() error {
-			return exporter.ExportSpans(ctx, spans)
-		})
+	var (
+		wg    sync.WaitGroup
+		errMu sync.Mutex
+		errs  = make([]error, 0, len(m.exporters))
+	)
+
+	for _, exporter := range m.exporters {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			if err := exporter.ExportSpans(ctx, spans); err != nil {
```

Comment:
I didn't use a `errgroup` for these, but we could if we want the slightly easier UX. But once we have go1.25 as minimum version, these can be simpified using `WaitGroup.Go`; https://pkg.go.dev/sync#WaitGroup.Go